src/Metadata/Resource/Factory/ResourceMetadataFactoryTrait.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace ApiPlatform\Core\Metadata\Resource\Factory;
+
+use ApiPlatform\Core\Exception\ResourceClassNotFoundException;
+use ApiPlatform\Core\Metadata\Resource\Operation;
```

Comment:
Actually we should be able to remove [this whole portion of code](https://github.com/api-platform/core/blob/a7441dfcf2349eb94cd590541409b5095efb51fd/src/Metadata/Resource/Factory/AnnotationResourceMetadataFactory.php#L102-L143). Operations are handled through [OperationResourceMetadataFactory](https://github.com/api-platform/core/blob/a7441dfcf2349eb94cd590541409b5095efb51fd/src/Metadata/Resource/Factory/OperationResourceMetadataFactory.php).